integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -720,7 +720,10 @@ void shouldCallAuthorizeAsDocAppClient(String uiLocales) throws JOSEException, P
                 equalTo(List.of("[\"P2.Cl.Cm\",\"Cl.Cm\"]")));
         assertThat(
                 clientSession.getVtrList(),
-                equalTo(List.of(VectorOfTrust.of(MEDIUM_LEVEL, LevelOfConfidence.MEDIUM_LEVEL))));
+                equalTo(
+                        List.of(
+                                VectorOfTrust.of(MEDIUM_LEVEL, LevelOfConfidence.MEDIUM_LEVEL),
+                                new VectorOfTrust(MEDIUM_LEVEL))));
```

Comment:
Note: This is expected to fail once ATO-248 implements the logic change to stop lists that have both identity and non-identity vectors